<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets.qrc">
    <normaloff>:/icons/img/network-idle.png</normaloff>:/icons/img/network-idle.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="sitesTableView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="threadsLabel">
        <property name="text">
         <string>Threads</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="threadsSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeoutLabel">
        <property name="text">
         <string>Timeout (sec)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timeoutSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="../assets.qrc">
          <normaloff>:/icons/img/media-playback-start.png</normaloff>:/icons/img/media-playback-start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="../assets.qrc">
          <normaloff>:/icons/img/media-playback-stop.png</normaloff>:/icons/img/media-playback-stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonTest">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="importUrlsAction"/>
    <addaction name="exportResultsAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="clearTableAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="importUrlsAction"/>
   <addaction name="exportResultsAction"/>
   <addaction name="separator"/>
   <addaction name="clearTableAction"/>
   <addaction name="separator"/>
   <addaction name="aboutAction"/>
   <addaction name="separator"/>
   <addaction name="quitAction"/>
  </widget>
  <action name="importUrlsAction">
   <property name="icon">
    <iconset resource="../assets.qrc">
     <normaloff>:/icons/img/document-open.png</normaloff>:/icons/img/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Import URLs</string>
   </property>
  </action>
  <action name="exportResultsAction">
   <property name="icon">
    <iconset resource="../assets.qrc">
     <normaloff>:/icons/img/document-save.png</normaloff>:/icons/img/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Export results</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="../assets.qrc">
     <normaloff>:/icons/img/system-log-out.png</normaloff>:/icons/img/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="clearTableAction">
   <property name="icon">
    <iconset resource="../assets.qrc">
     <normaloff>:/icons/img/edit-clear.png</normaloff>:/icons/img/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear table</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="../assets.qrc">
     <normaloff>:/icons/img/help-browser.png</normaloff>:/icons/img/help-browser.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets.qrc"/>
 </resources>
 <connections/>
</ui>
